cloudnet/src/main/java/de/dytanic/cloudnet/command/commands/CommandCreate.java

```diff
@@ -196,6 +197,9 @@ private static void listAndStartServices(ICommandSender sender, Collection<Servi
                                     Integer.parseInt(properties.get("memory")) : processConfiguration.getMaxHeapMemorySize(),
                             new ArrayList<>(properties.containsKey("jvmOptions") ?
                                     Arrays.asList(properties.get("jvmOptions").split(";")) :
+                                    processConfiguration.getJvmOptions()),
+                            new ArrayList<>(properties.containsKey("processParameters") ?
+                                    Arrays.asList(properties.get("processParameters").split(";")) :
                                     processConfiguration.getJvmOptions())
```

Comment:
JVM options as the process parameters?

Reply:
im the copy paste master xD
i will fix it now ^^

Reply:
fixed